CHANGELOG.md

```diff
@@ -1,3 +1,70 @@
+# 79.1.0
+
+This minor release increases our snapshot testing coverage and introduces a migration flag for
+including `UIAccessibilityTraitButton` in MDCButton's accessibilityTraits.
+
+While we do not expect the new MDCButton behavior to be breaking, you will temporarily be able to
```

Comment:
I don't think this phrasing is correct. The flag defaults to `YES` and clients can opt-in to the new behavior by setting it to `NO`.  This is a non-breaking, non-risky change.